Update continuous-delivery-faqs.md (#6508)
harness/developer-hub: kb/continuous-delivery/continuous-delivery-faqs.md

Yes, users can now store the terraform plan on the delegate and leverage it in the apply step. This now bypasses the restriction to store the plan in a secrets manager and let users store it locally.
This feature is behind the feature flag, `CDS_STORE_TERRAFORM_PLAN_FILE_LOCALLY_ON_DELEGATE`. Harness Delegate version 827xx or later is required for this feature.
Go to [Store Terraform Plan on Harness Delegate](https://developer.harness.io/docs/continuous-delivery/cd-infrastructure/terraform-infra/run-a-terraform-plan-with-the-terraform-plan-step/#store-terraform-plan-on-harness-delegate) and [Demo Video](https://www.loom.com/share/bc5a4f382d584b228b4ea2c82eb94a7c?sid=b9fac5c3-c11b-4f50-acff-f4fd2b3cc83a) for more information.

> #### Harness is pulling old helm dependencies that are not in the Chart.yaml

Reviewer: ```suggestion
#### Harness is pulling old Helm dependencies that are not in the Chart.yaml.
```

> Check the service config and if you have configured the override, and check if the chart is getting pulled from the correct location, and we also list the manifest so review if that looks correct.

Reviewer: ```suggestion
Check the following:
* Service configuration.
* Whether you have configured the override. 
* Whether the chart is getting pulled from the correct location.
* List of manifests.
```

#### How to get stage.Selected env?

> You can use environment variables like <+env.name> or <+env.identifier>, so it will resolve to the env used for that stage.

Reviewer: ```suggestion
You can use environment variables like <+env.name> or <+env.identifier>, so that it resolves to the environment used for that stage.
```

Yes, users can now store the terraform plan on the delegate and leverage it in the apply step. This now bypasses the restriction to store the plan in a secrets manager and let users store it locally.
This feature is behind the feature flag, `CDS_STORE_TERRAFORM_PLAN_FILE_LOCALLY_ON_DELEGATE`. Harness Delegate version 827xx or later is required for this feature.
Go to [Store Terraform Plan on Harness Delegate](https://developer.harness.io/docs/continuous-delivery/cd-infrastructure/terraform-infra/run-a-terraform-plan-with-the-terraform-plan-step/#store-terraform-plan-on-harness-delegate) and [Demo Video](https://www.loom.com/share/bc5a4f382d584b228b4ea2c82eb94a7c?sid=b9fac5c3-c11b-4f50-acff-f4fd2b3cc83a) for more information.

#### Harness is pulling old Helm dependencies that are not in the Chart.yaml.

Check the following:
* Service configuration.
* Whether you have configured the override. 
* Whether the chart is getting pulled from the correct location.
* List of manifests.

#### How to get stage.Selected env?

> You can use environment variables like <+env.name> or <+env.identifier>, so that it resolves to the environment used for that stage.

Reviewer: ```suggestion
You can use environment variables like `<+env.name>` or `<+env.identifier>`, so that it resolves to the environment used for that stage.
```
